lms/envs/common.py

```diff
@@ -340,9 +340,6 @@
     # Enable OpenBadge support. See the BADGR_* settings later in this file.
     'ENABLE_OPENBADGES': False,
 
-    # The block types to disable need to be specified in "x block disable config" in django admin.
-    'ENABLE_DISABLING_XBLOCK_TYPES': True,
```

Comment:
Note to reviewers: this is no "feature flag" anymore for disabling xblocks. Since it is a blacklist operation, if one does not wish to use the feature, one simply shouldn't add anything to the XBlockConfiguration model.